examples/prometheus/prometheus.yaml

```diff
@@ -48,10 +53,12 @@ objects:
     name: prometheus
     namespace: "${NAMESPACE}"
   spec:
+    host: "${PROMETHEUS_HOSTNAME}"
```

Comment:
fixed in https://github.com/openshift/origin/pull/15078.
closing this PR to avoid confusion